Implements [OPTIONAL] Swap Left/Right hand, changes [OPTIONAL] default keybinds to accomodate

tontyGH opened the pull request:
## About The Pull Request

- Adds **OPTIONAL** keybinds that lets you select a specific hand
- Alters **OPTIONAL** default keybinds as such:
  - Q: Swap to Right Hand (replacing Drop item)
  - E: Swap to Left Hand (replacing Quick equip)
  - R: Quick equip (replacing Toggle Throw Mode)
  - ShiftR: Quick equip belt
  - X: Drop item (replacing Swap hands)
  - Swap hands is Unbound

## Why It's Good For The Game

Swapping hands has always been somewhat awkward. Since it depends on the currently selected hand, swapping your hands requires you to look away from what you're doing and mentally process what you're holding and what you have selected. In the heat of the moment (like a fight) it's really easy to forget which hand you have selected and do something you did not want to do (like accidentally hug your target instead of attack them). This makes it so that, at the very least, you only need to check what you're holding.

The **OPTIONAL** default keybinds **THAT ARE OPTIONAL YOU DON'T HAVE TO USE THEM** were altered so that new players use this new feature, hopefully alleviating some of the hiccups from learning hand swapping.
## Changelog

:cl:
qol: Added [OPTIONAL] keybinds to for Swap Left/Right hands. Check your keybinds. [NOT OPTIONAL]
/:cl:

tontyGH committed:
implements the keybind

tontyGH committed:
makes it the default controls

Loafin34 commented:
what about 4-handed trait? that thing is where I usually get mixed up and lose where my hand slot is.

Mothblocks commented:
Update the wiki please after merge

tontyGH committed:
Returns Use to being bound to Z
I was hesitating on this so much that ultimately I decided to just honor legacy controls since they both are equally good candidates for different reasons.

If someone presses R instead of E by accident, putting it away is probably safer than using it (if they're holding a grenade for example

deathrobotpunch commented:
> Q/E will swap between two hands of a given set, though there is no way to swap between different sets. I was considering adding a keybind to swap between hand sets because of this, but it's super niche...

it’s probably possible by making Q and e progress left and right individually every time you press it, so if you have four hands and you want the leftmost hand you press q twice

Mothblocks reviewed (changes requested):
You'll need to change the swap_hands tutorial to be whatever the right hand key will be https://github.com/tgstation/tgstation/blob/e4f7898b82c7ae971ce3326e79cb13abd650e438/code/modules/tutorials/tutorials/switch_hands.dm#L54-L57

tontyGH committed:
updates swap hands tutorial
